src/hdinsightonaks/azext_hdinsightonaks/aaz/latest/hdinsightonaks/__cmd_group.py

```diff
@@ -0,0 +1,24 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command_group(
+    "hdinsightonaks",
```

Comment:
I think it's better use '-' to separate the words in the group name?

Reply:
If those commands belong to `aks`, you can also use `aks hdinsight` as the top level command group.

Reply:
These commands don't belong to aks. We are HDInisght service.
We already have one module named `az hdinsight`, now we onboard a new service named hdinsightonaks, so we want to use the az hdinsightonaks.

We checked the existing modules that some use the `-` but some not.

is it possible to use the `hdinsightonaks`? Thanks

Reply:
It's different between module/extension name and command group name. You can use any name for the module/extension name. But for the command and command group name, please follow this [naming guideline](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#command-naming-and-behavior)